src/main/java/example/nosql/CloudantClientMgr.java

```diff
@@ -19,8 +19,8 @@
 
 	private static String databaseName = "sample_nosql_db";
 
-	private static String user = null;
-	private static String password = null;
+	private static String user = "REPLACE_WITH_CLOUDANT_USERNAME";
```

Comment:
I know this isn't new, but would be nice if we could read the credentials from a file when running locally like the other cloudant boilerplates do so we're not asking users to put their credentials in source code.